lib/main.dart

```diff
@@ -10,9 +10,6 @@ class MyApp extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return const MaterialApp(
-      debugShowCheckedModeBanner: false,
-      home: HomePage(),
-    );
+    return SearchScreen();
   }
```

Comment:
your main should only have the main entry point for your app, not a specific search screen or anything else. If there is only one screen for the app, then don't name it something else, define it such that someone understands that this is supposed to be the main entry point to the app.

Reply:
This is temporary. This app will later have more pages so I will be implementing routes.

Reply:
Then we should setup routing in the beginning only, otherwise we will have to clear more mess